ocp_resources/performance_profile.py

```diff
@@ -0,0 +1,103 @@
+from ocp_resources.resource import Resource
+
+
+class PerformanceProfile(Resource):
+    api_group = Resource.ApiGroup.PERFORMANCE_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        additional_kernel_args=None,
+        cpu=None,
+        globally_disable_irq_load_balancing=False,
+        hugepages=None,
+        machine_config_label=None,
+        machine_config_pool_selector=None,
+        net=None,
+        node_selector=None,
+        numa=None,
+        real_time_kernel=None,
+        workload_hints=None,
+        **kwargs,
+    ):
+        """
+        PerformanceProfile object. API reference:
+        https://docs.openshift.com/container-platform/4.13/rest_api/node_apis/performanceprofile-performance-openshift-io-v2.html#performanceprofile-performance-openshift-io-v2
+
+        Args (in .spec):
+            additional_kernel_args (array of strings): Additional kernel arguments.
+            cpu (dict): Set of CPU related parameters. For example:
+                {
+                    "isolated": "4-39,44-79",
+                    "reserved": "0-3,40-43",
+                }
+            globally_disable_irq_load_balancing (bool, default: False): Toggles whether IRQ load balancing will be
+                disabled for the Isolated CPU set.
+            hugepages (dict): Defines a set of huge pages related parameters. It is possible to set huge pages with
```

Comment:
`t is possible to set huge pages with ...` is not relevant here; the code does not do that; this is related to the api. please add what key-value are expected